code/game/objects/items/devices/illegalradio.dm

```diff
@@ -148,15 +160,42 @@
 	dat = jointext(dat,"") //Optimize BYOND's shittiness by making "dat" actually a list of strings and join it all together afterwards! Yes, I'm serious, this is actually a big deal
 	return dat
 
-/obj/item/device/illegalradio/proc/generate_buyer_menu(mob/user)
+	
+	
+/obj/item/device/illegalradio/proc/generate_delivery_menu(mob/user, var/datum/black_market_item/item)
+	var/dat = list()
+	dat += "<B><font size=5>["The Black Market"]</font></B><BR>"
```

Comment:
Why are you doing this?

Reply:
Doing what? Having a proc return a string? I'm not sure what you mean.

Reply:
no, why are you doing a string inside the variable brackets?

Escape character is \ if you were trying to make it say "The Black Market" in quotes.